sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi_simple.c

```diff
@@ -388,18 +349,53 @@ static inline void stabilization_indi_calc_cmd(int32_t indi_commands[], struct I
  * @param in_flight not used
  * @param rate_control rate control enabled, otherwise attitude control
  */
-void stabilization_indi_run(bool in_flight __attribute__((unused)), bool rate_control)
+void stabilization_indi_attitude_run(bool in_flight __attribute__((unused)), struct Int32Quat quat_sp) // how does quat_sp is set when using RC?
 {
   /* attitude error                          */
-  struct Int32Quat att_err;
+  struct Int32Quat att_err; 
```

Comment:
remove tailing whitespace

Reply:
may I ask why? the compiler will just pass through it, no?

Reply:
yes, indeed, it is only to follow the coding style: http://docs.paparazziuav.org/latest/stylec.html
you can also use the `./fix_code_style.sh` script at the root of the project with files to format as arguments.